.github/workflows/verify_base_branch.yaml

```diff
@@ -0,0 +1,40 @@
+name: Pull Request base branch verification
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, edited]
+
+jobs:
+  test-base-branch:
+    runs-on: ubuntu-latest
```

Comment:
Provide a descriptive name for the job.
![image](https://user-images.githubusercontent.com/3183926/133576330-05dde794-4ef6-4d72-97f9-c9181c155012.png)

```suggestion
  test-base-branch:
    name: Pull Request base branch verification
    runs-on: ubuntu-latest
```